packages/sync-actions/src/product-actions.js

```diff
@@ -490,6 +490,8 @@ export function actionsMapCategories(diff) {
 
 export function actionsMapCategoryOrderHints(diff) {
   if (!diff.categoryOrderHints) return []
+  // Ignore this pattern as its means no changes happened [{},0,0]
```

Comment:
The `[{},0,0]` in this comment confused me a bit and I didn't see this anywhere else in this project. But we can keep i!